pom.xml

```diff
@@ -207,6 +207,11 @@
                 <artifactId>testng</artifactId>
                 <version>${testng.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>${org.apache.commons.version}</version>
+            </dependency>
```

Comment:
currently this dependency has used in two places. in a test case (TenantDeadLetterChannelTestCase) and test resource (AndesAdminClient). as for now in product-mb test resources are compiled in compile scope and test cases are compiled in test scope. Therefore, to do this change we have to change all test resource scopes from compile to test. This can be done as a separate improvement task to product-mb.